manifests/config.pp

```diff
@@ -2,7 +2,7 @@
 #
 # This class is called from uchiwa
 #
-class uchiwa::config {
+class uchiwa::config inherits uchiwa {
```

Comment:
I'm ok with inheritance being used *inside* a module. It sure helps a lot when you have a lot of params on the init class and you want them all available in the config class.